src/suffer.cpp

```diff
@@ -1169,8 +1169,13 @@ void Character::suffer_from_bad_bionics()
     // Negative bionics effects
     if( has_bionic( bio_dis_shock ) && get_power_level() > 9_kJ && one_turn_in( 2_hours ) &&
         !has_effect( effect_narcosis ) ) {
-        add_msg_if_player( m_bad, _( "You suffer a painful electrical discharge!" ) );
-        mod_pain( 1 );
+            if( !has_trait( trait_NOPAIN ) ) {
+                add_msg_if_player( m_bad, _( "You suffer a painful electrical discharge!" ) );
+                mod_pain( 1 );
```

Comment:
`mod_pain` includes this check, so all this is really doing is adding a different message.
That isn't bad, I just want to make it clear to everyone reviewing.

Reply:
Dang, I should have tracked that function down. Now I feel kinda silly.